pydisdrometer/DropSizeDistribution.py

```diff
@@ -133,30 +131,29 @@ def __init__(self, reader, time_start = None, location=None,
         self.fields = {}
         self.time_start = time_start
 
-        self.m_w_dict = {'0C': refractive.m_w_0C ,'10C': refractive.m_w_10C , '20C':  refractive.m_w_20C  }
-
-        self.m_w = self.m_w_dict[scattering_temp]
-
         self.numt = len(reader.time['data'])
         location = {}
 
         if location:
             self.location = {'latitude': location[0], 'longitude': location[1]}
 
-    def change_scattering_temperature(self, scattering_temp='10C'):
+        self.set_scattering_temperature_and_frequency()
+
+    def set_scattering_temperature_and_frequency(self, scattering_temp=10, scattering_frequency = 9.7e9):
         ''' Change the scattering temperature. After use, re-run calculate_radar_parameters
-        to see the effect this has on the parameters
+        to see the effect this has on the parameters. Temperatures are in Celsius. Defaults to 10C X-band.
 
         Parameters
         ----------
-        temp: optional, string
+        temp: optional, float
             String of temperature to scatter at. Choice of ("0C","10C","20C").
```

Comment:
Should change the docstring to indicate the new float.
Something like: "Scattering temperature [C]"?